core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -1723,6 +1723,17 @@ public String toString() {
           .setIsHidden(true)
           .setScope(Scope.MASTER)
           .build();
+  public static final PropertyKey MASTER_SKIP_ROOT_ACL_CHECK =
+      new Builder(Name.MASTER_SKIP_ROOT_ACL_CHECK)
+          .setDefaultValue(false)
+          .setDescription("Skip root directory ACL check when restoring from backup. This is to "
+              + "allow users to restore a backup from a different cluster onto their current one "
+              + "without having to recreate the different clusters owner user.")
+          .setConsistencyCheckLevel(ConsistencyCheckLevel.ENFORCE)
+          .setScope(Scope.MASTER)
+          .setIsHidden(true)
+          .setIgnoredSiteProperty(true)
```

Comment:
What does this do?

Reply:
Prevents the property from being set in alluxio-site.properties. This property should be set explicitly in alluxio-env.sh or in the command line with -D since it is not expected to be used in standard operating procedure.